### PR TITLE
Update docs for plugin submission form

### PR DESCRIPTION
**Pull request by Ukochka**
**What this PR does / why we need it**:
Update screenshot of plugin submission form to include 2 new fields: Testing guidance and Provisioning provided
Update description of this screenshot

### CODE REVIEW

File: docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Follow these steps to publish your plugin for the first time.
 1. In the left menu, under Org Settings, click **My Plugins**.
 1. Click **Submit New Plugin**. The Create Plugin Submission dialog box appears.
 
-   ![Submit plugin.](/img/plugins-submission-create2.png)
+   ![Submit plugin.](/img/plugins-submission-create.png)
```
Comment (xnyo) [inline docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md:40]: The image returns a 404. The correct URL is:

```suggestion
   ![Submit plugin.](/img/plugin-submission-create.png)
```

Also the old image (`img/plugins-submission-create2.png`) is not used anymore and should be deleted

**Review by xnyo — COMMENTED** [review]
Text changes are good but I found an issue with the image while testing :eyes:

File: docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md [hunk 2]
```diff
@@ -47,6 +47,8 @@ Follow these steps to publish your plugin for the first time.
    - **URL:** A URL that points to a ZIP archive of your packaged plugin.
    - **Source code URL:** A URL that points to a public Git repository or ZIP archive of your complete plugin source code.
    - **SHA1:** The SHA1 hash of the plugin specified by the **URL**.
+   - **Testing guidance:** An overview covering the installation, configuration, and usage of your plugin.
+   - **Provisioning provided for test environment:** It indicates that you've added provisioning functionality to your plugin.
```
Comment (sympatheticmoose) [inline docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md:51]:
```suggestion
   - **Provisioning provided for test environment:** Check this box if you have [configured provisioning](./provide-test-environment.md).
```

File: docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md (hunk 1 above)
Comment (sympatheticmoose) [reply docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md:40]: think this is fixed now?

**Review by sympatheticmoose — APPROVED** [review]
LGTM thanks for the quick turnaround 🙌 one suggestion on wording.

File: docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md (hunk 1 above)
Comment (xnyo) [reply docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md:40]: Yep the link has been fixed :raised_hands: 

However the old image `img/plugins-submission-create2.png` should be deleted.

**Pull request merged**